_posts/2022-08-03-using-h3-hexes.md

```diff
@@ -0,0 +1,126 @@
+
+Because [hexagons are the bestagons](https://www.youtube.com/watch?v=thOifuHs6eY), many people create hex grids that cover their area of interest and then aggregate their point data to these hexes.
+
+However, these custom hex grids are usually project-specific and **don't conform to any sort of common grid**, making it difficult to pull in other hexed data. And while hexagons tile a plane beautifully, you can't tesselate them to create a perfect larger hexagon, which means **it's difficult to create a larger or smaller grid** that relates to your original grid. And as anyone who's tried to aggregate a ton of points into a ton of hexagons can tell you, **spatial joins can be slow**.
+
+## H3: The Solution
+{: .text-left}
+
+Uber created and open-sourced the [H3 geospatial indexing system](https://h3geo.org/) to solve these problems. The grid used by H3 is a mathematically-defined series of hexagons arranged in tiers or resolutions such that each hexagon _logically_ completely contains seven child hexagons that are _geographically_ mostly contained by the larger hexagon.
+
+![H3 Hexagons at various scales]({% link images/h3_hexes.png %})
+{: .flex .flex--center}
+
+### The Basics
+{: .text-left}
+
+To understand how H3 solves these problems, we need to understand how it works. Coming from the GIS world, we think of the "point in polygon" problem spatially: search to see if the given point is inside the given area. To understand H3, we need to shift our thinking. **H3 takes a lat/long and assigns an ID that corresponds to a mathematically-defined hex.** There's nothing inherently spatial about it, it's all just math. The output is just a number.
+
+### Common Grids
+{: .text-left}
+
+As an open-source system, H3 has become a sort of de facto standard. Anyone can use it, and anyone can take someone else's data with H3 grid ids and compare it with their own. While it's been around for a few years, there's a definite "flavor of the month" aspect at play so it remains to be seen how widely it will be adopted.
+
+However, even if it doesn't become the overwhelming global standard, the H3 system can still be useful for **standardizing analyses across projects within an organization.** If we at UGRC do a project for one state agency with the grid, it would be easy to pull that data into another project down the road.
+
+### Scaling Grid Resolution
+{: .text-left}
+
+It is mathematically impossible<sup>1</sup> to create a larger hexagon from smaller hexagons, which makes creating a perfect _geographical_ hierarchy of hexes impossible. However, H3 makes a _logical_ hierarchy by rotating the larger, parent hex to cover as much of its seven children as possible, as visible in the image above. The H3 [indexing page](https://h3geo.org/docs/highlights/indexing) has more detail on how this works.
```

Comment:
**thought**: if this is a foot note kram down supports this in markdown. https://kramdown.gettalong.org/syntax.html#footnotes

Reply:
Cool, implemented!